recipes/libjpeg-turbo/all/conanfile.py

```diff
@@ -123,11 +123,13 @@ def package(self):
 
         self.copy("license*", src=self._source_subfolder, dst="licenses", ignore_case=True, keep_path=False)
 
+    def _collect_libs(self):
```

Comment:
This method is only used once. For next time I'd include this logic inside the package_info() to avoid extra code.